src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -5272,6 +5286,14 @@
           PowerSource:
               BatChargeLevel:
                   - Ok
+          WiFiNetworkDiagnostics:
+              WiFiVersionType:
+                  - 80211a
+                  - 80211b
+                  - 80211g
+                  - 80211n
+                  - 80211ac
+                  - 80211ax
```

Comment:
Stupid question but I'm wondering why 80211 is here when it is being removed everywhere else?

Reply:
If you look at line 5222, you'll see this is in the "deprecated" section, and so old values are put in here so auto-generated things can properly map old to new, for deprecation macros.

Reply:
Had a similar question in another PR and got an answer there as well, thanks for clarifying.